roles/ceph-mds/tasks/non_containerized.yml

```diff
@@ -20,6 +20,16 @@
     - mds_group_name in group_names
     - ansible_os_family == 'RedHat'
 
+- name: install suse ceph-mds package
+  package:
+    name: "ceph-mds"
+    state: "{{ (upgrade_ceph_packages|bool) | ternary('latest','present') }}"
+  register: result
+  until: result is succeeded
+  when:
+    - mds_group_name in group_names
+    - ansible_os_family == 'Suse'
```

Comment:
the package name is the same for RedHat and Suse, why not using the task above and use this condition:
`when: ansible_os_family in ['Suse', 'RedHat']`

Reply:
Sure, I'll change that. I was just following the "one task per OS" approach used before...